pytrack/core/py_track.py

```diff
@@ -29,19 +27,9 @@
 class PyTrackParent:
```

Comment:
class doc string

Reply:
solved in 2175398a95c5813644aef5da0d41bc8ce2df9d21